src/main/types.js

```diff
@@ -974,9 +1018,9 @@ export class VendorConnector {
 
     /**
      * Supervise a call
-     * @param {SupervisedCallInfo} call Call to be supervised
+     * @param {PhoneCall} call Call to be supervised
      */
-     superviseCall(supervisedCallInfo) {
+     superviseCall(call) {
```

Comment:
this should be reverted
should stay supervisedCallInfo

Reply:
Thanks for catching that. Updated